views/amp-access/article.html

```diff
@@ -79,7 +83,7 @@
     <main role="main">
     <article>
 
-      <section id="meter-notif" amp-access="access" class="meter-section" data-z-depth="2">
+      <section id="meter-notif" amp-access="access OR error" class="meter-section" data-z-depth="2">
```

Comment:
This section is visible because we recommend to show content in case of an error?